lib/notifier.rb

```diff
@@ -53,10 +55,16 @@ def run
   end
 
   def pages
-    JSON.parse(HTTP.get(@pages_url)).map { |data|
-      data['url'] = get_absolute_url(data['url'])
-      Page.new(data)
-    }
+    begin
+      JSON.parse(HTTP.get(@pages_url)).map { |data|
+        data['url'] = get_absolute_url(data['url'])
+        Page.new(data)
+      }
+    rescue => exception
+      warn "Notifier: could not get pages for tech docs at #{@pages_url}"
+      warn exception.message
+      return nil
```

Comment:
Might want to return `[]` here instead of `nil` - code further down calls `pages` and assumes it will be able to call enumerable methods (like `.reject`) on the result (see line 71 / 72).